install_config/topics/full_rhv_inventory.adoc

```diff
@@ -0,0 +1,112 @@
+openshift_enable_service_catalog=False
+
+app_dns_prefix=apps
+public_hosted_zone=example.com
+load_balancer_hostname=lb.{{public_hosted_zone}}
+openshift_master_cluster_hostname="{{ load_balancer_hostname }}"
+openshift_master_cluster_public_hostname=openshift-master.{{ public_hosted_zone }}
+openshift_master_default_subdomain="{{ app_dns_prefix }}.{{ public_hosted_zone }}"
+openshift_public_hostname="{{openshift_master_cluster_public_hostname}}"
+
+# Pod Networking
+os_sdn_network_plugin_name=redhat/openshift-ovs-networkpolicy
+
+# Registry
+openshift_hosted_registry_storage_kind=glusterfs
+
+# Authentication (example here creates one user, myuser with password changeme)
+openshift_master_identity_providers="[{'name': 'htpasswd_auth', 'login': 'True', 'challenge': 'True', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]"
+openshift_master_htpasswd_users={'myuser': '$apr1$zAhyA9Ko$rBxBOwAwwtRuuaw8OtCwH0'}
+
+# Red Hat Subscription Management
+# (vault_ variables are meant to be provided by Ansible Vault to protect passwords)
+rhsub_pool=Red Hat OpenShift Container Platform*
+rhsub_user="{{ vault_rhsub_user }}"
+rhsub_password="{{ vault_rhsub_password }}"
+
+# Docker and extra file system setup
+container_runtime_docker_storage_setup_device=/dev/vdb
+container_runtime_docker_storage_type=overlay2
+openshift_docker_use_system_container=False
+openshift_node_local_quota_per_fsgroup=512Mi
+openshift_use_system_containers=False
+
+[masters:vars]
+# Use the container_runtime section of the prerequisites playbook to set up extra file systems
+container_runtime_extra_storage="[{'device': '/dev/vdc', 'path': '/var/lib/origin/openshift.local.volumes', 'options': 'gquota', 'filesystem': 'xfs', 'format': 'True'}, {'device': '/dev/vdd', 'path': '/var/lib/etcd', 'hosts': 'masters', 'filesystem': 'xfs', 'format': 'True'}]"
+
+[nodes:vars]
+# Use the container_runtime section of the prerequisites playbook to set up extra file systems
+container_runtime_extra_storage="[{'device': '/dev/vdc', 'path': '/var/lib/origin/openshift.local.volumes', 'options': 'gquota', 'filesystem': 'xfs', 'format': 'True'}]"
+
+[masters]
+master0.example.com
+master1.example.com
+master2.example.com
+
+[etcd]
+master0.example.com
+master1.example.com
+master2.example.com
+
+[infras]
+infra0.example.com
+infra1.example.com
+infra2.example.com
+
+[glusterfs_registry]
+infra0.example.com
+infra1.example.com
+infra2.example.com
+
+[lb]
+lb.example.com
+
+[nodes]
+master0.example.com openshift_node_group_name=node-config-master
+master1.example.com openshift_node_group_name=node-config-master
+master2.example.com openshift_node_group_name=node-config-master
+infra0.example.com openshift_node_group_name=node-config-infra glusterfs_devices="['/dev/vdd']"
+infra1.example.com openshift_node_group_name=node-config-infra glusterfs_devices="['/dev/vdd']"
+infra2.example.com openshift_node_group_name=node-config-infra glusterfs_devices="['/dev/vdd']"
```

Comment:
The `glusterfs_devices=...` should be put under  `[glusterfs_registry]` section, see the https://docs.openshift.com/container-platform/3.10/install_config/persistent_storage/persistent_storage_glusterfs.html#install-example-registry